public/urls.json

```diff
@@ -5,9 +5,12 @@
   "edd-about-contact": "https://www.edd.ca.gov/About_EDD/Contact_EDD.htm",
   "edd-about-privacy-policy": "https://www.edd.ca.gov/About_EDD/Privacy_Policy.htm",
   "edd-ca-gov": "https://edd.ca.gov",
-  "bpo-log-in": "https://portal.edd.ca.gov/WebApp/Login",
-  "bpo-log-out": "https://portal.edd.ca.gov/WebApp/Logout",
+  "bpo-prefix": "https://portal.edd.ca.gov/WebApp",
+  "bpo-login": "https://portal.edd.ca.gov/WebApp/Login",
+  "bpo-logout": "https://portal.edd.ca.gov/WebApp/Logout",
+  "uio-prefix": "https://uio.edd.ca.gov/UIO",
```

Comment:
Using prefixes + replace seems cleaner to me than suffixes + concatenate-- it means everything in urls.json is still a URL, and I feel like it's harder to make a mistake on where the prefix/suffix begin and end. @rocketnova lmk if I'm missing something

Reply:
Oh yah this looks good too. My main concern is to avoid hard coding the prefixes into `getUrl()`.